cid/utils.py

```diff
@@ -50,3 +53,55 @@ def get_boto_client(service_name, **kwargs):
     except Exception as e:
         logger.debug(e, stack_info=True)
         raise
+
+def get_parameter(param_name, message, choices=None, default=None, none_as_disabled=False, template_variables={}, break_on_ctrl_c=True):
+    """ 
+    Check if parameters are provided in the command line and if not, ask user 
+
+    :param message: text message for user
+    :param choices: a list or dict for choice. None for text entry
+    :param default: a default text template
+    :param none_as_disabled: if True and choices is a dict, all choices with None as a value will be disabled
+    :param template_variables: a dict with varibles for template
+    :param break_on_ctrl_c: if True, exit() if user pressed CTRL+C
+
+    :returns: a value choosed by user or provided in command line    
+    """
+    if param_name in params:
+        value = params[param_name]
+        print(f'Using {param_name}={value}, from parameters')
+        return value.format(**template_variables) 
+
+    if choices is not None:
+        if isinstance(choices, dict):
+            _choices = []
+            for key, value in choices.items():
+                print(key, value)
+                _choices.append(
+                    questionary.Choice(
+                        title=key,
+                        value=value,
+                        disabled=True if (none_as_disabled and value is None) else False,
+                    )
+                )
+                choices = _choices
+        print()
```

Comment:
Needs to be removed

Reply:
the issue is that we have print(..., end='') in several places, so the prompt will be in the same line. We need to remove  end='' or leave the print here